src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/boss_razorgore.cpp

```diff
@@ -23,48 +23,49 @@
 
 enum Say
 {
-    SAY_EGGS_BROKEN1        = 0,
-    SAY_EGGS_BROKEN2        = 1,
-    SAY_EGGS_BROKEN3        = 2,
-    SAY_DEATH               = 3,
+    SAY_EGGS_BROKEN1            = 0,
+    SAY_EGGS_BROKEN2            = 1,
+    SAY_EGGS_BROKEN3            = 2,
+    SAY_DEATH                   = 3,
 
-    EMOTE_TROOPS_RETREAT    = 0
+    EMOTE_TROOPS_RETREAT        = 0
 };
 
 enum Spells
 {
-    SPELL_MINDCONTROL       = 19832,
-    SPELL_EGG_DESTROY       = 19873,
-    SPELL_MIND_EXHAUSTION   = 23958,
+    SPELL_MINDCONTROL           = 19832,
+    SPELL_MINDCONTROL_VISUAL    = 45537,
```

Comment:
SPELL_MINDCONTROL_VISUAL    = 23014,

Reply:
Wrong spell 45537, correct should be 23014

Reply:
It's the same spell as Grethok casts.